examples/07-Train-traditional-ML-models-using-the-Merlin-Models-API.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Are these (precisions@10, auc) the default evaluation metrics? What other metrics are available and how does one specify them? (A link to a page might suffice).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>these are the default metrics available in our implementation, no other metrics are available nor can this list of metrics be customized</p>